src/app/cube/new-home/help/build-program/build-program.component.scss

```diff
@@ -0,0 +1,197 @@
+@import 'vars.scss';
+
+.wrapper {
+    margin: 0px 35px;
+    font-family: 'Source Sans Pro', sans-serif;
+
+    .info {
+        font-size: 20px;
+        font-weight: 600;
+    }
+    
+    .framework {
+        font-size: 18px;
+        margin: 0 10px 10px 0;
+        padding: 10px 24px;
+        border: 2px solid #b3b3b3;
+        border-radius: 3px;
+        font-family: 'Inter', sans-serif;
+
+        &:hover {
+            border: 2px solid $light-blue;
+            opacity: 80%;
+        }
+
+        &:active {
+            border: 2px solid $light-blue;
+            background-color: #8caff143;
+            opacity: 100%;
+        }
+    }
+
+    .search {
+        .input {
+            cursor: text;
+            background-color: white;
+            padding: 7px;
+            margin-right: 12px;
+            border: 2px solid #dfdfdf;
```

Comment:
I'm pretty sure I've added this value to vars in a few of my PR's, $secondary-white, and there are a couple more hex values in this file that could be put in vars